guides/source/active_record_migrations.md

```diff
@@ -5,35 +5,43 @@ Active Record Migrations
 
 Migrations are a feature of Active Record that allows you to evolve your
 database schema over time. Rather than write schema modifications in pure SQL,
-migrations allow you to use a Ruby DSL to describe changes to your tables.
+migrations allow you to use a Ruby Domain Specific Language (DSL) to describe
+changes to your tables.
 
 After reading this guide, you will know:
 
-* The generators you can use to create them.
-* The methods Active Record provides to manipulate your database.
-* The rails commands that manipulate migrations and your schema.
+* Which generators you can use to create migrations.
+* Which methods Active Record provides to manipulate your database.
+* How to change existing migrations and update your schema.
 * How migrations relate to `schema.rb`.
+* How to maintain referential integrity.
 
 --------------------------------------------------------------------------------
 
 Migration Overview
 ------------------
 
-Migrations are a convenient way to
-[alter your database schema over time](https://en.wikipedia.org/wiki/Schema_migration)
-in a consistent way. They use a Ruby DSL so that you don't have to
-write SQL by hand, allowing your schema and changes to be database independent.
+Migrations are a convenient way to [evolve your database schema over
+time](https://en.wikipedia.org/wiki/Schema_migration) in a consistent way. They
+use a Ruby [DSL](https://en.wikipedia.org/wiki/Domain-specific_language) so that
+you don't have to write SQL by hand, allowing your schema and changes to be
+database independent. We recommend that you read the guides for [Active Record
+Basics](active_record_basics.html) and the [Active Record
+Associations](association_basics.html) to learn more about some of the concepts
+mentioned here.
 
 You can think of each migration as being a new 'version' of the database. A
 schema starts off with nothing in it, and each migration modifies it to add or
 remove tables, columns, or entries. Active Record knows how to update your
```

Comment:
I'd probably cut "entries" and maybe change this to "add or remove things like tables, columns, or indexes"

Reply:
Or give it even more space to explain what the "rails way" to add or remove records using migrations is, since not everyone agrees, and people may have found conflicting information when googeling.